coverage/report-lcov/lcov-report/index.html

```diff
@@ -20,24 +20,24 @@
     </h1>
     <div class='clearfix'>
       <div class='fl pad1y space-right2'>
-        <span class="strong">100% </span>
+        <span class="strong">90% </span>
```

Comment:
You should remove it the coverage folder from git's memory. Use `git rm --cached ./coverage/*`